tests/publiccloud/prepare_tools.pm

```diff
@@ -75,12 +75,14 @@ sub run {
     # Install ec2imgutils
     install_in_venv('ec2imgutils', 'ec2uploadimg');
     assert_script_run("curl " . data_url('publiccloud/ec2utils.conf') . " -o /root/.ec2utils.conf");
-    # TODO: revert to 'ec2uploadimg --version' when it will be fixed
-    record_info('ec2imgutils', 'ec2uploadimg:' . script_output('ec2uploadimg -h'));
+    record_info('ec2imgutils', 'ec2uploadimg:' . script_output('ec2uploadimg --version'));
 
     # Install Azure cli
     install_in_venv('azure-cli', 'az');
-    record_info('Azure', script_output('az -v'));
+    my $azure_error = '/tmp/azure_error';
+    record_info('Azure', script_output('az -v 2>' . $azure_error));
```

Comment:
Why not use a simple `assert_script_run('az -v')`? Since we are using the serial terminal, the output should be in the serial txt file on error or are you regulary checking the output via openQA?

Reply:
Maybe we can add `proceed_on_failure => 1` as we check the error file later?

Reply:
@grisu48 we had the case that it print someting on `stderr` but the command exit with 0

@pdostal I wouldn't do `proceed_on_failure => 1 ` that thing is so bad implemented, that baybe they exit with `!= 0` but don't write something to `stderr` :)

Reply:
> Why not use a simple `assert_script_run('az -v')`? Since we are using the serial terminal, the output should be in the serial txt file on error or are you regulary checking the output via openQA?

this implementation allow simultaneously : 
1. get actual output of `az -v`  in `record_info` so you can see it in Details tab and don't need to do several extra actions to read it 
2. verify output for potential errors 

In your version I am not sure that we cover both goals . Another problem that I am not sure if `az ` return non-zero exit code in case of Warning

Reply:
I understand - then it's a clear :+1:  from my side